source/manual/alerts/slo-rate-checks-account.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-account-tech"
+title: GOV.UK Account SLO error rate exceeded 1% in 10 minute window
```

Comment:
1% is part of the SLO, if we change the SLO then we don't really want to have to update the title of this page too. 
 Also, it's good if this title is close to the Icinga alert titles, so it's easier to find in search.

How about:

```
SLO Error Rate on GOV.UK Accounts has exceeded an acceptable threshold across the last 10 minutes
```